py2many/cli.py

```diff
@@ -366,15 +360,89 @@ def _process_once(settings, filename, outdir, env=None):
 
         if restore_cwd:
             os.chdir(restore_cwd)
+    except Exception as e:
+        print(f"Error: Could not format: {output_path}")
+        print(f"Due to: {e.__class__.__name__} {e}")
+        return False
 
     return True
 
 
+def _get_output_path(filename, ext, outdir):
```

Comment:
could you put this above `_format_one` so that its inner code is no longer being moved - git will just mark the lines around it as changed.